pkgs/misc/tmux-plugins/default.nix

```diff
@@ -184,6 +184,17 @@ in rec {
     };
   };
 
+  dracula = mkDerivation rec {
+    pluginName = "dracula";
+    version = "unstable-2020-10-08";
+    src = pkgs.fetchFromGitHub {
+      owner = "dracula";
+      repo = "tmux";
+      rev = "8cf490184259bd423571415d4d73a20d8fb1850b";
+      sha256 = "1x7v55z1ljq0757b6xsy1rxp598k3p53pcwx0pasb7ispbs1jjir";
+    };
```

Comment:
:books: Would it be possible to add the meta attributes, you can look at [resurrect](https://github.com/NixOS/nixpkgs/blob/440c5808a18980909ca2eb349d4819d266967e49/pkgs/misc/tmux-plugins/default.nix#L294-L319) as an example (omit `longDescription` if unavailable). You will also need to add yourself as a maintainer of this package.